docs/package.json

```diff
@@ -67,5 +69,17 @@
     "setupFiles": [
       "<rootDir>/loadershim.js"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,json,jsx,md,mdx,ts,tsx}": [
+      "npm run fmt",
+      "npm run fmt-check",
```

Comment:
Do we need this? `fmt-check` was/is just meant to be a way to check in CI if things were formatted (if we want to put this in CI). But if you're running `npm run fmt` on staged things, then `npm run fmt-check` might be redundant. Thoughts?

Reply:
Prettier couldn't format some of the `mdx` files correctly - I resolved these issues manually. I added the `fmt-check` in case we run into this issue again. Probably redundant now though as it's unlikely we will.